src/templates/layouts/vetCenter/index.test.tsx

```diff
@@ -220,4 +220,97 @@ describe('VetCenter with valid data', () => {
+      const dataWithDifferentOfficialName = {
+        ...mockData,
+        title: 'Pittsburgh Vet Center',
+        officialName: 'Pittsburgh Veterans Center for Readjustment',
+      }
+
+      render(<VetCenter {...dataWithDifferentOfficialName} />)
+
+      // Should render the "Also called" text
+      expect(
+        screen.getByText(
+          'Also called the Pittsburgh Veterans Center for Readjustment'
+        )
+      ).toBeInTheDocument()
+
+      // Should have the h1 with aria-describedby pointing to the "Also called" element
+      const heading = screen.getByRole('heading', { level: 1 })
+      expect(heading).toHaveAttribute('aria-describedby', 'vet-center-title')
+
+      // Should have the correct ID on the "Also called" paragraph
+      const alsoCalledElement = screen.getByText(
+        'Also called the Pittsburgh Veterans Center for Readjustment'
+      )
+      expect(alsoCalledElement).toHaveAttribute('id', 'vet-center-title')
+    })
+
+    test('does not render "Also called" text when officialName is same as title', () => {
+      const dataWithSameNames = {
+        ...mockData,
+        title: 'Pittsburgh Vet Center',
+        officialName: 'Pittsburgh Vet Center',
+      }
+
+      render(<VetCenter {...dataWithSameNames} />)
+
+      // Should not render the "Also called" text
+      expect(screen.queryByText(/Also called the/)).not.toBeInTheDocument()
+
+      // h1 should not have aria-describedby attribute
+      const heading = screen.getByRole('heading', { level: 1 })
+      expect(heading).not.toHaveAttribute('aria-describedby')
+    })
+
+    test('does not render "Also called" text when officialName is null', () => {
+      const dataWithNullOfficialName = {
+        ...mockData,
+        title: 'Pittsburgh Vet Center',
+        officialName: null,
+      }
+
+      render(<VetCenter {...dataWithNullOfficialName} />)
+
+      // Should not render the "Also called" text
+      expect(screen.queryByText(/Also called the/)).not.toBeInTheDocument()
+
+      // h1 should not have aria-describedby attribute
+      const heading = screen.getByRole('heading', { level: 1 })
+      expect(heading).not.toHaveAttribute('aria-describedby')
+    })
+
+    test('does not render "Also called" text when officialName is empty string', () => {
+      const dataWithEmptyOfficialName = {
+        ...mockData,
+        title: 'Pittsburgh Vet Center',
+        officialName: '',
+      }
+
+      render(<VetCenter {...dataWithEmptyOfficialName} />)
+
+      // Should not render the "Also called" text
+      expect(screen.queryByText(/Also called the/)).not.toBeInTheDocument()
+
+      // h1 should not have aria-describedby attribute
+      const heading = screen.getByRole('heading', { level: 1 })
+      expect(heading).not.toHaveAttribute('aria-describedby')
+    })
+
+    test('handles case where title is null but officialName exists', () => {
```

Comment:
```suggestion
    test('does not render "Also called" when title is null but officialName exists', () => {
```

To keep consistency with the other tests.